lib/engine/tile.rb

```diff
@@ -444,11 +447,6 @@ def revenue_to_render
       @revenue_to_render ||= @stops.map(&:revenue_to_render)
     end
 
```

Comment:
why did you change this

Reply:
This was messing up code I was doing to generalise being able to put unlabeled tiles on labeled hexes, but I ended up throwing it away because my approach was getting too messy. The removal got lost on the revert. I've put it back